src/rf/apps/workers/status_updates.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+from datetime import datetime
+
+from apps.core.models import LayerImage, Layer
+import apps.core.enums as enums
+
+
+TIMEOUT_SECONDS = (60 * 10)  # 10 Minutes.
+
+JOB_VALIDATE = [
+    'ObjectCreated:CompleteMultipartUpload',
+    'ObjectCreated:Put',
+    'ObjectCreated:Post'
+]
+JOB_HANDOFF = 'emr_handoff'
+JOB_TIMEOUT = 'timeout'
+
+TIMEOUT_DELAY = 60
+
+
+def mark_image_valid(s3_uuid):
+    try:
+        image = LayerImage.objects.get(s3_uuid=s3_uuid)
+        image.status = enums.STATUS_VALID
+        image.save()
+        return True
+    except (ValueError, LayerImage.DoesNotExist):
+        return False
+
+
+def mark_image_invalid(s3_uuid, error_message):
+    try:
+        image = LayerImage.objects.get(s3_uuid=s3_uuid)
+        image.status = enums.STATUS_INVALID
+        image.error = error_message
+        image.save()
+        layer_id = image.layer_id
+        update_layer_status(layer_id, enums.STATUS_FAILED, error_message)
+        return True
+    except (ValueError, LayerImage.DoesNotExist):
```

Comment:
Consider casting arguments to the correct type so we don't have to catch `ValueError` throughout.